man/foreman-debug.8.asciidoc

```diff
@@ -39,17 +39,55 @@ OPTIONS
 
 The following options are available:
 
+  --db-tables
+          Include selected tables from Foreman database
+  --db-dump
+          Include full database dump (generates big tarball)
+  --access-logs
+          Include HTTP access logs
+
   -d DIR  Directory to place the tarball in (default /tmp/foreman-XYZ)
   -g      Skip generic info (CPU, memory, firewall etc.)
   -a      Do not generate a tarball from the resulting directory
-  -m NUM  Maximum lines to keep for each file (default 5000)
+  -m NUM  Maximum lines to keep for each file (default 5000, 0 - no limit)
   -j PRG  Filter with provided program when creating a tarball
-  -p      Additionally print all passwords which are being filtered
+  -p      Additionally print password patterns being filtered out
   -q      Quiet mode
   -v      Verbose mode
   -u      Upload tarball
   -h      Shows this message
 
+SECURITY
+--------
+
+By default, foreman-debug does not include any data that are sensitive, such as
+passwords, tokens, host names or IP addresses. It is possible tho to explicitly
```

Comment:
1) "possible _though_ to"

2) Is this true?  By default it includes production.log, which contains IP addresses and hostnames, so I wouldn't claim this.